substrate/frame/contracts/src/benchmarking/mod.rs

```diff
@@ -1048,38 +1152,51 @@ benchmarks! {
 				params: vec![ValueType::I32, ValueType::I32],
 				return_type: Some(ValueType::I32),
 			}],
-			call_body: Some(body::repeated(r, &[
-				Instruction::I32Const(0), // value_ptr
-				Instruction::I32Const(0), // value_len
-				Instruction::Call(0),
-				Instruction::Drop,
-			])),
-			.. Default::default()
+			call_body: Some(body::repeated(
+				r,
+				&[
+					Instruction::I32Const(0), // value_ptr
+					Instruction::I32Const(0), // value_len
+					Instruction::Call(0),
+					Instruction::Drop,
+				],
+			)),
+			..Default::default()
 		});
 		let instance = Contract::<T>::new(code, vec![])?;
-	}: {
-		<Contracts<T>>::bare_call(
-			instance.caller,
-			instance.account_id,
-			0u32.into(),
-			Weight::MAX,
-			None,
-			vec![],
-			DebugInfo::UnsafeDebug,
-			CollectEvents::Skip,
-			Determinism::Enforced,
-		)
-		.result?;
+
+		#[block]
+		{
+			<Contracts<T>>::bare_call(
+				instance.caller,
+				instance.account_id,
+				0u32.into(),
+				Weight::MAX,
+				None,
+				vec![],
+				DebugInfo::UnsafeDebug,
+				CollectEvents::Skip,
+				Determinism::Enforced,
+			)
+			.result?;
+		}
+		Ok(())
 	}
 
-	seal_debug_message_per_byte {
-		// Vary size of input in bytes up to maximum allowed contract memory
-		// or maximum allowed debug buffer size, whichever is less.
-		let i in 0 .. (T::Schedule::get().limits.memory_pages * 64 * 1024).min(T::MaxDebugBufferLen::get());
+	// Vary size of input in bytes up to maximum allowed contract memory
+	// or maximum allowed debug buffer size, whichever is less.
+	#[benchmark]
+	#[rustfmt::skip]
```

Comment:
can be removed once 
this is merged https://github.com/paritytech/polkadot-sdk/pull/3598